src/PrestaShopBundle/Form/Admin/Improve/Design/Pages/CmsPageCategoryType.php

```diff
@@ -73,8 +79,12 @@ public function __construct(array $allCmsCategories, $isShopFeatureEnabled)
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $invalidCharactersForCatalogLabel = $this->trans('Invalid characters:', 'Admin.Global') . '<>;=#{}';
+        $invalidCharactersForNameLabel = $this->trans('Invalid characters:', 'Admin.Global') . '<>={}';
```

Comment:
```suggestion
        $invalidCharactersForCatalogLabel = $this->trans('Invalid characters:', 'Admin.Global') . '<>;=#{}';
        $invalidCharactersForNameLabel = $this->trans('Invalid characters:', 'Admin.Global') . '<>={}';
```

Could you move the string `'<>={}'` as constant ?

Reply:
Added contants from TypedRegexValidator